dagger/dagger_test.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os/exec"
+	"testing"
+)
+
+var chain string
+
+func init() {
+	flag.StringVar(&chain, "chain", "", "Specify the blockchain chain to test against")
```

Comment:
i'm not sure we can pass arguments to the tests like this (maybe I'm wrong).  We could also use environment variables.